src/services/queue-service.js

```diff
@@ -30,11 +38,31 @@ export async function getUserQueue() {
   } finally {
     checkInProgress = false;
     scheduleNextQueuePull(response);
+    setQueueAPIVersion(response);
   }
 
   return response;
 }
 
+function setQueueAPIVersion(response) {
+  if (response && response.headers) {
+    var queueVersion = response.headers["x-cio-queue-version"];
```

Comment:
Not a blocker to merge but I was wondering if the "x-cio-queue-version" should be a constant. That was the one hard coded value that made me pause. It isn't in multiple places or anything. I gave it another thought as something that isn't hardcoded thinking it was important and shouldn't change.

Reply:
If we were using it in many places yeah, but in this case I think its easier to read.